aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/Translator.java

```diff
@@ -111,15 +116,27 @@ static RestoreDbClusterToPointInTimeRequest restoreDbClusterToPointInTimeRequest
                 .kmsKeyId(model.getKmsKeyId())
                 .networkType(model.getNetworkType())
                 .publiclyAccessible(model.getPubliclyAccessible())
-                .restoreType(model.getRestoreType())
                 .scalingConfiguration(translateScalingConfigurationToSdk(model.getScalingConfiguration()))
                 .serverlessV2ScalingConfiguration(translateServerlessV2ScalingConfiguration(model.getServerlessV2ScalingConfiguration()))
                 .sourceDBClusterIdentifier(model.getSourceDBClusterIdentifier())
                 .storageType(model.getStorageType())
                 .tags(Tagging.translateTagsToSdk(tagSet))
                 .useLatestRestorableTime(model.getUseLatestRestorableTime())
-                .vpcSecurityGroupIds(model.getVpcSecurityGroupIds())
-                .build();
+                .vpcSecurityGroupIds(model.getVpcSecurityGroupIds());
+
+        if(StringUtils.hasValue(model.getRestoreToTime())
+                && BooleanUtils.isNotTrue(model.getUseLatestRestorableTime())
+                && !COPY_ON_WRITE.equalsIgnoreCase(model.getRestoreType())) {
```

Comment:
The last condition sort of looks like a conflicting parameter set rather than a trivial attribute toggle. From this statement we might conclude that if COW is set as the current restore type, restoreToTime should not be set at all and could either be rejected by the schema validator or by throwing an explicit invalid request exception.

Reply:
Done.